README.md

```diff
@@ -44,3 +44,23 @@ These are the arguments that apply to all commands:
     ```bash
     odf --context=$(oc config current-context) [commands]
     ```
+
+## Testing
+
+1. Clone the repository
```

Comment:
actually, in Markdown syntax we add `1.`  only in the numbering and it's auto-match otherwise in markdown syntax it through error
```yaml
MD029/ol-prefix: Ordered list item prefix [Expected: 1; Actual: 2; Style: 1/1/1]markdownlint[MD029](https://github.com/DavidAnson/markdownlint/blob/v0.32.1/doc/md029.md)
```